docs/conf.py

```diff
@@ -108,6 +109,38 @@
 plot_working_directory = "."
 plot_html_show_source_link = False
 
+# ----------------------------------------------------------------------------------
+# Redirects
+# ----------------------------------------------------------------------------------
+
+_inlined_apis = [
+    ("qiskit_addon_obp", "backpropagate"),
+    ("qiskit_addon_obp.utils.operations", "apply_op_to"),
+    ("qiskit_addon_obp.utils.operations", "to_global_op"),
+    ("qiskit_addon_obp.utils.operations", "reduce_op"),
```

Comment:
I suppose we must also add the missing function `apply_reset_to` here?
```suggestion
    ("qiskit_addon_obp.utils.operations", "reduce_op"),
    ("qiskit_addon_obp.utils.operations", "apply_reset_to"),
```

Reply:
Negative because `apply_reset_to` was not previously in the documentation so there is no need to set up a redirect